geomstats/learning/pca.py

```diff
@@ -108,15 +107,15 @@ def __init__(self, metric, n_components=None, copy=True,
         self.iterated_power = iterated_power
         self.random_state = random_state
 
-    def fit(self, X,
-            base_point=None, point_type='vector', y=None):
```

Comment:
Just to be sure: The `y` argument ist to comply with the scikit-learn interface to PCA?

Reply:
Actually, if we want this to be compatible with sickit-learn (and the Pipeline module in particular, for which `pipeline.fit(X, y)` propagates through all the steps of the pipeline without specifying `y=y`), I think `y=None` needs to be the second argument of `fit`.

Reply:
Yes, it is for compliance. Thanks @nguigs for the arguments order, I have put y as second parameter now.